app/assets/stylesheets/application.css

```diff
@@ -199,6 +199,7 @@ body {
 	.job {
 		border-bottom: 1px solid #ccc;
 		padding: 6px;
+    line-height: 20px;
```

Comment:
I'd change #L225 to height 15px other than making all the lines bigger.

Why do you prefer this solution?

|.job.remote_icon img - 15px| current broken state |
|---|---|
|![screen shot 2016-04-29 at 12 37 30](https://cloud.githubusercontent.com/assets/3641733/14915511/958ade6e-0e0b-11e6-9700-fc1d1c76dc2b.png)|![screen shot 2016-04-29 at 12 36 46](https://cloud.githubusercontent.com/assets/3641733/14915512/958fc19a-0e0b-11e6-8f36-62533e647003.png)|


https://github.com/rubyireland/rubyjobs2/pull/16/files#diff-fb7484bc2f56c263954da6fc44982eeaR226